### PR TITLE
📖  Add documentation files for 1.0 Bento components

### PR DESCRIPTION
**Pull request by caroqliu**
For the listed components, this PR does two things to its documentation:
- Copy its existing `.md` file to `0.1/`
- Modify its root `.md` file to refer to its `1.0` version - including adding `### Migrating from 0.1` if applicable and removing `## Validation`

Components affected:
- `amp-date-display` #30189
- `amp-inline-gallery` #28284
- `amp-instagram` #30172
- `amp-lightbox` #31052
- `amp-selector` #28282
- `amp-stream-gallery` #28284
- `amp-timeago` #29246
- `amp-video`
- `amp-youtube` #30443

### CODE REVIEW

File: extensions/amp-inline-gallery/amp-inline-gallery.md [hunk 1]
```diff
@@ -102,6 +113,8 @@ The pagination indicator location defaults to underneath the carousel. Adding th
 
 Displays the pagination indicator as inset, overlaying the carousel itself. When using `inset`, you should give the `<amp-inline-gallery-pagination>` element `layout="nodisplay"`.
 
+Note that when specifying an `inset` element, the `<amp-inline-gallery-pagination>` will be positioned with `bottom: 0` to its parent element. You may target and overwrite this to position the element over the carousel in the case where they may be other elements (such as `<amp-inline-gallery-thumbnails>`) directly below it.
```
Comment (CrystalOnScript) [inline extensions/amp-inline-gallery/amp-inline-gallery.md:116]:
```suggestion
Specify an `inset` element positions `<amp-inline-gallery-pagination>` at `bottom: 0` to its parent element. You may overwrite this to position the element over the carousel if there are other elements, such as `<amp-inline-gallery-thumbnails>`, directly below it.
```

File: extensions/amp-lightbox/amp-lightbox.md [hunk 2]
```diff
@@ -43,85 +43,13 @@ component. To show a gallery of images in a lightbox, you can use
 
 [/tip]
 
-[filter formats="ads"]
+### Migrating from 0.1
 
-### Using `amp-lightbox` in AMPHTML ads <a name="a4a"></a>
+Unlike `0.1`, the experimental `1.0` version of `amp-lightbox` employs the following changes:
```
Comment (CrystalOnScript) [inline extensions/amp-lightbox/amp-lightbox.md:48]:
```suggestion
The experimental `1.0` version of `amp-lightbox` employs the following differences from version `0.1`:
```

File: extensions/amp-youtube/amp-youtube.md [hunk 3]
```diff
@@ -56,6 +57,10 @@ With the responsive layout, the width and height from the example should yield c
 </amp-youtube>
 ```
 
+### Migrating from 0.1
+
+Unlike `0.1`, the experimental `1.0` version does not automatically convert `data-param-controls` to `controls`, and takes `controls` directly as an attribute.
```
Comment (CrystalOnScript) [inline extensions/amp-youtube/amp-youtube.md:62]:
```suggestion
The experimental `1.0` version of `amp-youtube` does not automatically convert `data-param-controls` to `controls`. Instead it takes `controls` directly as an attribute.
```

**Review by CrystalOnScript — APPROVED** [review]
Looking good! Just a few nits for clarity and active voice. Thanks you!

File: extensions/amp-inline-gallery/amp-inline-gallery.md (hunk 1 above)
Comment (caroqliu) [reply extensions/amp-inline-gallery/amp-inline-gallery.md:116]: I think the rewording lost some of the original message in the first sentence, is this okay? :
```
The default CSS for an `<amp-inline-gallery-pagination>` element with `inset` specifies it at `bottom: 0` with respect to its parent element. 
```

Kept the second sentence you recommended.

File: extensions/amp-youtube/amp-youtube.md (hunk 3 above)
Comment (caroqliu) [reply extensions/amp-youtube/amp-youtube.md:62]: Updated, thanks!

File: extensions/amp-lightbox/amp-lightbox.md (hunk 2 above)
Comment (caroqliu) [reply extensions/amp-lightbox/amp-lightbox.md:48]: Updated!

File: extensions/amp-video/0.1/amp-video.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+</amp-video>
+```
+
+[/example]
+
+## Attributes
+
+### src
+
+Required if no <`<source>` children are present. Must be HTTPS.
```
Comment (alanorozco) [inline extensions/amp-video/0.1/amp-video.md:66]: Not sure if this is already present but

```suggestion
Required if no `<source>` children are present. Must be HTTPS.
```

**Review by alanorozco — APPROVED** [review]

**Pull request merged**